sql/table.go

```diff
@@ -77,8 +76,10 @@ func makeTableDesc(p *parser.CreateTable) (structured.TableDescriptor, error) {
 				}
 				if d.PrimaryKey {
 					index.Name = "primary"
+					desc.Indexes = append([]structured.IndexDescriptor{index}, desc.Indexes...)
```

Comment:
can you add a test that exercises this change? I believe all the tests you added only exercise the change below that validates the presence of a primary key.